frame/scheduler/src/lib.rs

```diff
@@ -93,12 +123,49 @@ struct ScheduledV1<Call, BlockNumber> {
 	maybe_periodic: Option<schedule::Period<BlockNumber>>,
 }
 
+// NOTE: use `Derivative` here until <https://github.com/rust-lang/rust/issues/26925> is fixed.
+// The problem is that `#[derive(Clone)]` requires `Clone` for ALL its generic types,
+// which does not make sense for `Get<>`.
```

Comment:
This means you should use `CloneNoBound`.

Reply:
Was looking for something like this. Thank you, sir!